src/commands/ExecCommand.js

```diff
@@ -0,0 +1,34 @@
+import ChildProcessUtilities from "../ChildProcessUtilities";
+import Command from "../Command";
+import async from "async";
+
+export default class ExecCommand extends Command {
+  initialize(callback) {
+    this.command = this.input[0];
+    this.args = this.input.slice(1);
+
+    if (!this.command) {
+      callback(new Error("You must specify which npm command to run."));
+      return;
+    }
+
+    callback(null, true);
+  }
+
+  execute(callback) {
+    async.parallelLimit(this.packages.map(pkg => cb => {
+      this.runCommandInPackage(pkg, cb);
+    }), 4, callback);
+  }
+
+  runCommandInPackage(pkg, callback) {
+    ChildProcessUtilities.exec(this.command, { cwd: pkg.location }, (err, stdout) => {
```

Comment:
This should be handling more than just a single argument. It needs to handle things like:

```
lerna exec ls -la subdirectory 
```

It'd probably be better if you exposed the entire `process.argv` on `this.argv` inside commands and then trim the `lerna exec` from the beginning simply passing the rest through.